pkg/deploy/predeploy.go

```diff
@@ -506,35 +502,29 @@ func (d *deployer) restartOldScalesets(ctx context.Context, resourceGroupName st
 	return nil
 }
 
-func (d *deployer) restartOldScaleset(ctx context.Context, vmssName string, resourceGroupName string) error {
-	var restartScript string
-	switch {
-	case strings.HasPrefix(vmssName, gatewayVMSSPrefix):
-		restartScript = gatewayRestartScript
```

Comment:
Can we also remove the `gatewayRestartScript` var at the beginning of the file?

Reply:
Done.